### PR TITLE
added front end styling to the form page and others

### PR DESCRIPTION
**Pull request by Chenxi96**
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have added thorough tests. (If tests are not applicable, explain why in the next section.)
- [x] Is there only 1 commit.

## Describe your changes
While working with chakraUI version 3, I found an issue with testing with Jest. It gave me an reference error, which is saying that the structuredClone is not defined. I noticed that version 3 was released last month (link: https://next.chakra-ui.com/blog/00-announcing-v3). There might be other issues that can pop up while dealing with the newest version of chakraUI. I made a decision to downgrade to version 2 to reduce to amount of debugging for now. Until there's more information on how to fix this issue will be using version 2 for now.

route: '/create'
localhost_3000_create

main page:
localhost_3000_

My changes:
1. Added chakraUI to style the front end.
2. Modified the Middleware to not run on the main page.

### CODE REVIEW

File: components/CreateFormPost.tsx
Comment (Chenxi96) [inline components/CreateFormPost.tsx:1]: styled the form post

**Review by alvinc21 — COMMENTED** [review]
Looks pretty good. Just remove Ngrok and the dev environment variable in favour of strictly using Prisma for the dev and prod environments. Also, pull requests that include front end changes should include a screen shot.

**Comment by Chenxi96** [comment]
> Looks pretty good. Just remove Ngrok and the dev environment variable in favour of strictly using Prisma for the dev and prod environments. Also, pull requests that include front end changes should include a screen shot.

File: README.md [hunk 1]
```diff
@@ -7,12 +7,6 @@
 2. Install dependencies with `npm install`.
 3. Install docker `https://docs.docker.com/get-docker/`
```
Comment (Chenxi96) [inline README.md:9]: removed ngrok in readme

**Review by alvinc21 — APPROVED** [review]

**Pull request merged**